app/lib/features/authentication/register/presentation/pages/register_page.dart

```diff
@@ -0,0 +1,127 @@
+part of '_pages.dart';
+
+class RegisterPage extends StatefulWidget {
+  const RegisterPage({super.key});
+
+  @override
+  State<RegisterPage> createState() => _RegisterPageState();
+}
+
+class _RegisterPageState extends State<RegisterPage> {
+  late final UserRegisterBloc _userRegisterBloc;
+  TextEditingController dateinput = TextEditingController();
+
+  String? selectedValue = null;
+
+  @override
+  void initState() {
+    _userRegisterBloc = get<UserRegisterBloc>();
+
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocListener<UserRegisterBloc, RegisterFormState>(
+        listener: (context, state) {
+      if (state is UserRegisterSuccessState) {
+        context.go('/main');
+      }
+    },
+    child: BlocBuilder<UserRegisterBloc, RegisterFormState>(
+        builder: (context, state) {
+          return Scaffold(
```

Comment:
please use `BebrasScaffold` to follow the standard
<img width="477" alt="image" src="https://github.com/ia-toki/bebras-pandai/assets/1970083/13b4ec9e-eb3d-402f-8199-528f51d91831">

also this
```
const SizedBox(
  height: 40,
),                
```